components/addFileButton/index.jsx

```diff
@@ -27,6 +27,9 @@ import PodLocationContext from "../../src/contexts/podLocationContext";
 import AlertContext from "../../src/contexts/alertContext";
 import ConfirmationDialogContext from "../../src/contexts/confirmationDialogContext";
 
+const TESTCAFE_ID_UPLOAD_BUTTON = "upload-file-button";
```

Comment:
I think we should the constants into one file, that way we get one place to look up used IDs.

Reply:
I don't agree - I think it'll be too much back and forth when developing. @kevin-inrupt's tool inspects the dom anyways, rather than browsing source code.